app/controllers/org_admin/sections_controller.rb

```diff
@@ -0,0 +1,148 @@
+module OrgAdmin
+  class SectionsController < ApplicationController
+    include Versionable
+    
+    respond_to :html
+    after_action :verify_authorized
+
+    # GET /org_admin/templates/[:template_id]/phases/[:phase_id]/sections
+    def index
+      authorize Section.new
+      phase = Phase.includes(:template, :sections).find(params[:phase_id])
+      edit = (current_user.can_modify_templates?  &&  (phase.template.org_id == current_user.org_id))
+      render partial: 'index', 
+        locals: { 
+          template: phase.template, 
+          phase: phase, 
+          sections: phase.sections, 
+          current_section: phase.sections.first,
+          current_tab: params[:r] || 'all-templates',
+          edit: edit 
+        }
+    end
+
+    # GET /org_admin/templates/[:template_id]/phases/[:phase_id]/sections/[:id]
+    def show
+      section = Section.includes(questions: [:annotations, :question_options]).find(params[:id])
+      authorize section
+      render partial: 'show', locals: { section: section }
+    end
+
+    # GET /org_admin/templates/[:template_id]/phases/[:phase_id]/sections/[:id]/edit
+    def edit
+      section = Section.includes({phase: :template}, questions: [:annotations, :question_options]).find(params[:id])
+      authorize section
+      render partial: 'edit', 
+        locals: { 
+          template: section.phase.template, 
+          phase: section.phase, 
+          section: section ,
+          current_tab: params[:r] || 'all-templates',
+          edit: true
+        }
+    end
+
+    # GET /org_admin/templates/[:template_id]/phases/[:phase_id]/sections/new
+    def new
+      phase = Phase.includes(:template, :sections).find(params[:phase_id])
+      section = Section.new(phase: phase, number: (phase.sections.length > 0 ? phase.sections.max{ |a, b| a.number <=> b.number }.number+1 : 1))
+      authorize section
+      render partial: 'new', 
+        locals: { 
+          template: phase.template,
+          phase: phase,
+          section: section,
+          current_tab: params[:r] || 'all-templates'
+        }
+    end
+
+    # POST /org_admin/templates/[:template_id]/phases/[:phase_id]/sections
+    def create
+      phase = Phase.includes(:template, :sections).find(params[:phase_id])
+      section = Section.new(section_params.merge({ phase_id: phase.id }))
+      authorize section
+      begin
+        section = get_new(section)
+      
+# TODO: update UI so that this comes in as part of the `section:` part of the params
+        section.description = params["section-desc"]
```

Comment:
now that we are refactoring why don't we use the description field instead of "section-desc" ?